lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAttachmentsAccTest.java

```diff
@@ -44,6 +47,7 @@
   @BeforeEach
   @WithAccessId(user = "admin")
   void setUp() throws Exception {
+    resetDb(false);
```

Comment:
why?

Reply:
our tests previously did not reset the db, so the introduction of a new test led to the failure of other tests, when executed sequentially
this line resets the state of the db
it seems costly, but leads to more consistent tests

the need for this line shows, that it could be time to rethink at least parts of our testing strategy